### DISCUSSION TITLE
gps telemetry

### DISCUSSION THREAD

**Discussion by landrieve** [EdgeTX/edgetx#2848] [Q&A] [answered]
hi. Why we dont have on gps telemetry the choice between NMEA an DSM with EDGETX. This choice is on OpenTX.
Thanks
luc

**Comment by raphaelcoeffic** [comment, marked as answer]
What is the DSM format? Did you mean DMS?

I believe you should find that setting in the "Radio Settings / GPS" settings.

**Reply by pfeerick** [reply]
If we're talking about a colorLCD radio (the questioner doesn't mention what radio this is in relation to), yes indeed:
![companion_hkPpHmDtcv](https://user-images.githubusercontent.com/5500713/206954311-ad0271ad-881a-4f6b-9b61-255d49a0f85d.gif)

**Comment by landrieve** [comment]
i have not any option for gps. i have edgetx 2.6.0 in french version

**Reply by pfeerick** [reply]
You need to upgrade to at least v2.7.x (i.e. v2.7.1) to get the GPS options or do a special build of 2.6 enabling GPS support then ;)

**Reply by landrieve** [reply]
Ok, i have upgrade my tx16s with 2.7.1 but i have not the gps option on it but i have it on compagnion 2.7.1 so  i have change the gps option with it and save in my radio but my problem is the same. the coordonate send by my gps on my rx hitec is false and my gps is good because the coordonate are good with my tx hitec flash 7.

**Reply by pfeerick** [reply]
Hm... not long before 2.8 was released  I know there is an issue with the GPS options being hidden (if you don't enable the internal GPS) so fixed it then, but didn't realise it was also in 2.7.1 (i.e. if you go to the hardware page, enable GPS on one of the AUX ports, you will get the GPS options back on the radio setup page - but turn the GPS option off again once you've made the changes so the TX doesn't get confused). Sorry! At least on 2.7 you don't need a special firmware build for the internal GPS support (which those options were mistakenly linked to - as they affect system settings as well as telemetry settings). 

But if this doesn't fix your issue - which is now that the coordinates are wrong? - which perhaps stems from something to do with telemetry and the MPM (I presume this is how you are connecting to your Hitec Rx?). Probably need a little more info about the setup in order to figure out what is wrong (and possibly someone who uses Hitec gear as well😆)
